[Ftr] update readme (#410)
Edirom/Edirom-Online: README.md

[![Build](https://github.com/Edirom/Edirom-Online/actions/workflows/docker-ci.yml/badge.svg?branch=develop&event=push)](https://github.com/Edirom/Edirom-Online/actions/workflows/docker-ci.yml) [![NFDI4C Registry](https://nfdi4culture.de/fileadmin/user_upload/registry/badges/nfdi4culturebadge.svg)](https://nfdi4culture.de/id/E3648) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)

<div align="center">
 
**[Show cases](https://github.com/Edirom/Edirom-Online#-Show-cases) • 
[Get started](https://github.com/Edirom/Edirom-Online#-Get-started) • 
[Dependencies](https://github.com/Edirom/Edirom-Online#-Dependencies) • 
[Roadmap](https://github.com/Edirom/Edirom-Online#-Roadmap) • 
[Contributing](https://github.com/Edirom/Edirom-Online#-Contributing) • 
[Get in touch](https://github.com/Edirom/Edirom-Online#-Get-in-touch) • 
[Code of Conduct](https://github.com/Edirom/Edirom-Online#-Code-of-Conduct) • 
[License](https://github.com/Edirom/Edirom-Online#-License)**

</div>

# Edirom-Online

> Edirom-Online is software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data. This allows the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.  

Reviewer: was this suggestion left out on purpose? Just to make sure it was not forgotten …

> Edirom-Online is software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data. This allows the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.  

Author: sorry, i added a commit after your suggestions and (adressing one aspect of your review) but in this area, appearently i did not already accepted your suggestions. github does not allow me to commit your suggeestion siincee it is marked "outdated". sorry for the confusion witth my "dazwischen-grätschen"-commit. i accept all of your suggestions. please just add them all to the code.

The Edirom idea was born in 2004 at [Musikwissenschaftliches Seminar Detmold/Paderborn] and even after several years of Edirom development, the success of Edirom based on the same core concepts as in the beginning continues with numerous projects using and developing Edirom tools and creating digital musical editions with this software. Edirom tools were originally developed by the project [Entwicklung von Werkzeugen für digitale Formen wissenschaftlich-kritischer Musikeditionen] (2006–2012) funded by the DFG. The development of Edirom is now maintained as a community effort while being strongly supported and accompanied by [Virtueller Forschungsverbund Edirom] (ViFE), primarily based at the [Paderborn University]. ViFE aims to provide tools for scholars working with digital texts and music, especially those involved in editing historical documents.

## Show cases

To get some practical insights, look at this projects and editions that already use Edirom-Online.

> **Webers Klarinettenquintett op.34**

Author: thanks, i just added a [commit](https://github.com/Edirom/Edirom-Online/pull/410/commits/76fb190eea40dca480222b72b879f364d1d2e92d) to address your ideas. i ensured to provide all "institutions" with the links that are named here already in the introduction.

[![Build](https://github.com/Edirom/Edirom-Online/actions/workflows/docker-ci.yml/badge.svg?branch=develop&event=push)](https://github.com/Edirom/Edirom-Online/actions/workflows/docker-ci.yml) [![NFDI4C Registry](https://nfdi4culture.de/fileadmin/user_upload/registry/badges/nfdi4culturebadge.svg)](https://nfdi4culture.de/id/E3648) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)

<div align="center">
 
**[Show cases](https://github.com/Edirom/Edirom-Online#-Show-cases) • 
[Get started](https://github.com/Edirom/Edirom-Online#-Get-started) • 
[Dependencies](https://github.com/Edirom/Edirom-Online#-Dependencies) • 
[Roadmap](https://github.com/Edirom/Edirom-Online#-Roadmap) • 
[Contributing](https://github.com/Edirom/Edirom-Online#-Contributing) • 
[Get in touch](https://github.com/Edirom/Edirom-Online#-Get-in-touch) • 
[Code of Conduct](https://github.com/Edirom/Edirom-Online#-Code-of-Conduct) • 
[License](https://github.com/Edirom/Edirom-Online#-License)**

</div>

# Edirom-Online

> Edirom-Online is software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data, that is visualized with [Verovio]. This allows the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.  

Reviewer: once more
 ```suggestion
Edirom-Online is a software for the **presentation and analysis of critical musical editions** in a digital format, particularly in the fields of musicology and philology. Edirom-Online supports various data formats commonly used in digital humanities, such as [TEI] (Text Encoding Initiative) for textual data and [MEI] (Music Encoding Initiative) for musical data, that is visualized with [Verovio]. This allows for the integration of different data formats, starting in the early days with texts, images and music and adding audio and even film within a single edition.  
```

git submodule update --init --recursive
```

### Building locally

For building Edirom Online you need *Sencha Cmd* installed on your system. You might want to refer to the [Sencha Cmd System Setup] section for more details.

Alternatively we recommend to use a container image for building, e.g. [bwbohl/sencha-cmd]

```bash
docker run --rm -it -v /ABSOLUTE/PATH/TO/YOUR/LOCAL/EDIROM-ONLINE/CLONE:/app --name ediBuild ghcr.io/bwbohl/sencha-cmd:latest
```

> When you have your system preapared with all Sencha Cmd prerequisites or you have your docker container running you have to execute a sencha build command through calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference] for details), either in your native shell or in the container shell, e.g.:

Author: feel free to shorten or split the sentences as you think it supports the reading flow :)

> When you have your system preapared with all Sencha Cmd prerequisites or you have your docker container running you have to execute a sencha build command through calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference] for details), either in your native shell or in the container shell, e.g.:

Reviewer: How about this?
```suggestion
When you have your system prepared with all Sencha Cmd prerequisites or you have your docker container running you are now set up to execute the sencha build command. Do this by calling the build script included in this repository with one of the sencha build-type options (please refer to [sencha app build reference] for details), either in your native shell or in the container shell, e.g.:
```
